### PR TITLE
feat: loader for Go 1.19

### LINKED ISSUE #9: Support go1.19
**Is your feature request related to a problem? Please describe.**

Compiling using go1.19 will failed with the following messages:
../../../../go/pkg/mod/github.com/cloudwego/frugal@v0.1.1/internal/loader/funcdata_invalid.go:27:9: panic("Unsupported Go version. Supported versions are: 1.15, 1.16, 1.17, 1.18") (no value) used as value

**Describe the solution you'd like**
support compile using go1.19

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by choleraehyq**
Signed-off-by: Cholerae Hu <choleraehyq@gmail.com>

#### What type of PR is this?

feat: support Go 1.19

runtime/runtime2.go:_func, runtime/symtab.go:moduledata and related structs haven't been changed during 1.19 cycle, so I simply change the build tag in loader to support Go 1.19.

#### What this PR does / why we need it (en: English/zh: Chinese):

en: because Go 1.19 has been released.
zh: 因为 Go 1.19 已经发布了。

#### Which issue(s) this PR fixes:

Fixes #9

### CODE REVIEW

**Comment by choleraehyq** [comment]
PTAL @simon0-o @chenzhuoyu

**Comment by xieyh11** [comment]
+1, when to merge?

File: internal/loader/funcdata_invalid.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// +build !go1.15 go1.19
+// +build !go1.15 go1.20
```
Comment (chenzhuoyu) [inline internal/loader/funcdata_invalid.go:1]: Please update the error messages below accordingly.

**Review by chenzhuoyu — CHANGES_REQUESTED** [review]
Can you verify that the following structures in Go 1.19 are identical to Go 1.18?

**Comment by choleraehyq** [comment]
@chenzhuoyu 我在 description 里提到了，列出的几个结构我 check 了下在 1.19 cycle 没有更新。你们可以再 double check 下。

File: internal/loader/funcdata_invalid.go (hunk 1 above)
Comment (choleraehyq) [reply internal/loader/funcdata_invalid.go:1]: done

**Review by chenzhuoyu — APPROVED** [review]

**Pull request merged**